test/uk/gov/hmrc/merchandiseinbaggage/controllers/RetrieveDeclarationControllerSpec.scala

```diff
@@ -37,6 +37,7 @@ class RetrieveDeclarationControllerSpec extends DeclarationJourneyControllerSpec
 
   val view: RetrieveDeclarationView = injector.instanceOf[RetrieveDeclarationView]
   val connector: MibConnector       = injector.instanceOf[MibConnector]
+  private val stub                  = injector.instanceOf[MibBackendStub]
```

Comment:
Once the changes in the `MibBackendStub` have been done, please revert these changes, thanks